rfcs/0001-meta-schedule-autotensorir.md

```diff
@@ -22,36 +22,48 @@
 
 ## 1. Summary
 
-This proposal introduces Meta Schedule: a probabilistic scheduling DSL on TIR that unifies the
+This proposal introduces Meta Schedule: a scheduling DSL on TIR that unifies the
 approaches of AutoTVM and Auto Scheduler (Ansor). Meta schedule provides a pragmatic way to define
 the space of automatic tuning, extensibility in terms of all possible TIR schedule primitives like
 tensorization and loop partitioning, and customizability on every layer of the automation system.
 
-Meta Schedule is our 3rd generation automatic scheduling system.
+Meta Schedule is the 3rd generation automatic scheduling system.
 
 ## 2. Motivation
 
 **Scheduling and design space.** In TVM TensorIR, optimization of a TensorIR program is done via a
-sequence of transformations. For example, we reorder loops for better locality and we tensorize for
+sequence of transformations. For example, reordering loops for better locality and tensorizing for
 specific hardware intrinsics. The process of invoking such a set of pre-defined transformations is
 called "**scheduling**", and each transformation is called a "**schedule primitive**". These
```

Comment:
That makes sense. I will introduce scheduling as a general concept that both TE and TensorIR uses to transform the IR into potentially more optimized form. There are subtle difference here, because TE relies on a schedule tree which doesn't generate TIR until being lowered (i.e. `tvm.lower` is called), while TensorIR scheduling directly transforms the IR without the indirect schedule tree.